doctr/models/recognition/master/base.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+import numpy as np
+from typing import List, Tuple
+from ....datasets import VOCABS, encode_sequences
+
+
+class _MASTER:
+
+    vocab: str = VOCABS['french']
+    max_length: int = 50
```

Comment:
I think it's a bad idea to assign values here. If the constructor doesn't do it, it will hold values that are not easily accessible to the developper.

I guess this was for typing issues ? If so you can type them without assigning a value:
```python
vocab: str
max_length: int
```